src/rgen/Microsoft.Macios.Transformer/TransformCommand.cs

```diff
@@ -0,0 +1,38 @@
+using System.CommandLine;
+
+namespace Microsoft.Macios.Transformer;
+
+public class TransformCommand : Command {
+
+	public TransformCommand () : base ("rgen-transform", "command to convert outdated bindings to be rgen compatible")
```

Comment:
Is this tool going to be distributed publicly? If so, then consider using localized strings now, it will save you effort later on.